src/main/java/com/example/java_tutorial/NorthwindOlingoService.java

```diff
@@ -0,0 +1,52 @@
+package com.example.java_tutorial;
+
+import com.sap.cloud.sdk.cloudplatform.connectivity.DestinationAccessor;
+import com.sap.cloud.sdk.cloudplatform.connectivity.HttpDestination;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.apache.olingo.client.api.ODataClient;
+import org.apache.olingo.client.api.communication.request.retrieve.ODataEntitySetRequest;
+import org.apache.olingo.client.api.domain.ClientEntitySet;
+import org.apache.olingo.client.core.ODataClientFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.util.MultiValueMap;
+
+@Service
+public class NorthwindOlingoService {
+  private static final String DEST = "Northwind";
+  private final ODataClient client = ODataClientFactory.getClient();
+
+  public List<Map<String, Object>> fetchEntitySet(
```

Comment:
Is it possible for List<EntitySetType> to be returned? 
In case it is product, the List of Products will be returned. In case Shippers -> List<Shippers>